### DISCUSSION TITLE
Getting NBA odds on player totals - Cannot find via API

### DISCUSSION THREAD

**Discussion by codelancers-sd** [cloudbet/docs#79] [Q&A]
'Hello,

I am trying to get get the odds on player totals via the API. I am finding this to be quite difficult. For example, if I want odds on the match winner, I can easily get that using basketball.moneyline. But how do I get access to player totals (e.g. Curry to score 30 points). I cannot find that under listed markets for that event. Would appreciate any help

**Comment by rosheekhan** [comment]
It seems that the API you're using might not provide access to player prop bets such as individual player totals (e.g., Curry to score 30 points). You may need to explore other sports betting APIs or platforms that specifically offer player prop bets for NBA games. Alternatively, you could reach out to the provider of the API you're using to inquire if they have plans to include player prop bets in their offerings.

**Comment by kgravenreuth** [comment]
Hi @codelancers-sd , @rosheekhan ,

As mentioned in our [Cloudbet API Docs](https://www.cloudbet.com/api/), we offer a specific list of markets as listed in the `markets.json` file at https://gist.github.com/kgravenreuth/6703e1e213aecac4d5728f2f699d34e7

You can find the list of basketball markets offered with the prefix `basketball.` from that json file. At the moment, yes such individual player prop bets are excluded, while we may consider adding these in the future. We will update this thread if or when such a decision is taken. 

Please let us know if you have any other questions regarding market availability, thanks!

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
